lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -65,7 +65,12 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL, $fail
   // aware of a user's northstar id. If it doesn't find one, we just grab it
   // from northstar directly.
   if (!$northstar_id) {
-    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    if ($user->uid) {
+      $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+    } else {
+      $northstar_user = dosomething_northstar_get_user($values['northstar_id']);
+      $user = user_load($northstar_user->drupal_id);
```

Comment:
the code here looks fine, but above we are using the global user, so I'm not sure what case we are in where we don't have a user from that call. 
I don't think there would ever be a case where we would have a northstar id, but not a drupal id